compiler/modules/nback.llvm/llvm.bal

```diff
@@ -653,3 +658,15 @@ function omitSpaceBefore(string word) returns boolean {
 function omitSpaceAfter(string word) returns boolean {
     return word == "(" || word == "{";
 }
+
+// Corresponds to LLVMContextRef
+public type Context readonly & record {||};
+
+// Corresponds to LLVMContextCreate
+public function contextCreate() returns Context {
+    return {};
+}
+
+// Corresponds to LLVMContextDispose
+public function contextDispose(Context context) {
```

Comment:
Please remove. This doesn't work with a GC-language.

Reply:
Fixed